common/djangoapps/entitlements/helpers.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
+
+import pytz
+import sys
+from django.conf import settings
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def is_entitlement_expired(entitlement):
+    """
+    Determines from the policy if an entitlement can be redeemed, if it has not passed the
+    expiration period of policy.expiration_period_days, and has not already been redeemed
+    """
+    utc = pytz.UTC
+    days_since_created = (datetime.utcnow().replace(tzinfo=utc) - entitlement.created).days
+    expiration_period = settings.ENTITLEMENTS_POLICY.get('expiration_period_days', sys.maxint)
```

Comment:
Should we fail loudly if this is not set for some reason? It seems like a pretty important setting to allow to be unset.

Reply:
This was an open question both @jlajoie and I had, He mentioned it in the PR description and I think we might want to have a quick Whiteboard in the morning to discuss the best approach here.